int32 shelf_num
int32 robot_id
string action
string shoe_info
---
bool success